x/ecocredit/server/testsuite/suite.go

```diff
@@ -1038,6 +1047,86 @@ func (s *IntegrationTestSuite) TestScenario() {
 	})
 	s.Require().Nil(err)
 	s.Require().Equal(expectedSellOrderIds, createSellOrder.SellOrderIds)
+	orderId1 := createSellOrder.SellOrderIds[0]
+	orderId2 := createSellOrder.SellOrderIds[1]
+
+	// now we buy these orders
+	buyerAcc := acc5
+	expectedTotalCost := sdk.NewInt64Coin(coinPrice.Denom, 26_086_420)
+	// this is the exact amount it should cost to purchase both orders
+	s.fundAccount(buyerAcc, sdk.Coins{expectedTotalCost})
+
+	buyerAccBefore := s.getAccountInfo(buyerAcc, batchDenom, coinPrice.Denom)
+	sellerAccBefore := s.getAccountInfo(acc3, batchDenom, coinPrice.Denom)
```

Comment:
Might be nice to set `acc3` to `seller` at some point as you have done with `buyerAcc`.

Reply:
updated with sellerAcc in https://github.com/regen-network/regen-ledger/pull/1079/commits/2f1e43c799bef2167b427702fa541a9ced42ef04